### DISCUSSION TITLE
Log Context Feedback

### DISCUSSION THREAD

**Discussion by svennergr** [grafana/grafana#66870] [Observability]
Ask a question or provide feedback on the Log Context UI.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a discussion, and we'd like discussions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request a feature.
